deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -603,7 +603,7 @@ <h4 class="panel-title pull-left pointer-cursor">
                             capacitySetting.baseimages = sorted.map(function(o) {
                                 return {
                                     value: o.id,
-                                    text: o.provider_name
+                                    text: o.provider_name + '[' + o.acceptance + ']'
```

Comment:
Will this be Accepted or Denied? May give some more user friendly name or only show accepted for CMP by default